src/main/java/com/iotiq/user/internal/UserRepository.java

```diff
@@ -13,5 +13,7 @@ public interface UserRepository extends JpaRepository<User, UUID>, JpaSpecificat
     Optional<User> findByPersonalInfoEmail(String email);
 
     boolean existsByAccountInfoUsername(String username);
+    int countUsersWithAccountInfoUsernameAndNotId(UUID id, String username);
```

Comment:
I think, we use "boolean existsByUsernameIgnoreCaseAndIdNot(String username, UUID id)"

Reply:
"boolean existsByAccountInfoUsernameAndIdIsNot(String username, UUID id)" has been used.